server/routes/notification.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  getNewNotifications,
+  getAllNotifications,
+  createNotification,
+  readNotifications,
+} = require("../controllers/notification");
+
+router.route("/:id").post(protect, createNotification);
```

Comment:
```suggestion
router.route("/").post(protect, createNotification);
```
Based on comment above

Reply:
Applied as mentioned above.